client/state/initial-state.js

```diff
@@ -73,7 +73,7 @@ export default function createReduxStoreFromPersistedInitialState( reduxStoreRea
 			.then( reduxStoreReady );
 	} else {
 		debug( 'persist-redux is not enabled, building state from scratch' );
-		reduxStoreReady( createReduxStore( getInitialServerState() ) );
+		reduxStoreReady( createReduxStore( deserialize( {} ) ) );
```

Comment:
On second thought, this is probably ok. Leave a comment that this is loading with server state only?